cmd/commonflags_test.go

```diff
@@ -0,0 +1,138 @@
+/*
+ Copyright 2020 Qiniu Cloud (qiniu.com)
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCoverModeFlagValue(t *testing.T) {
+	var tcs = []struct {
+		value         string
+		expectedValue interface{}
+	}{
+		{
+			value:         "",
+			expectedValue: "count",
+		},
+		{
+			value:         "set",
+			expectedValue: "set",
+		},
+		{
+			value:         "count",
+			expectedValue: "count",
+		},
+		{
+			value:         "atomic",
+			expectedValue: "atomic",
+		},
+		{
+			value:         "xxxxx",
+			expectedValue: "",
+		},
+		{
+			value:         "123333",
+			expectedValue: "",
+		},
+	}
+	for _, tc := range tcs {
+		mode := &CoverMode{}
+		mode.Set(tc.value)
+		actual := mode.String()
+		assert.Equal(t, actual, tc.expectedValue, fmt.Sprintf("check mode flag value failed, expected %s, got %s", tc.expectedValue, actual))
+	}
+}
+
+func TestCoverModeFlagError(t *testing.T) {
```

Comment:
no need to spilt into this two methods `TestCoverModeFlagValue` and `TestCoverModeFlagError`